lib/live_view_native_swift_ui/modifiers/text_input_and_output/keyboard_shortcut.ex

```diff
@@ -0,0 +1,29 @@
+defmodule LiveViewNativeSwiftUi.Modifiers.KeyboardShortcut do
+  use LiveViewNativePlatform.Modifier
+
+  alias LiveViewNativeSwiftUi.Types.EventModifier
+
+  modifier_schema "keyboard_shortcut" do
+    field :key, Ecto.Enum, values: ~w(
+      up_arrow
+      down_arrow
+      left_arrow
+      right_arrow
+      clear
+      delete
+      end
+      escap
+      home
+      page_up
+      page_down
+      return
+      space
+      tab
+      a b c d e f g h i j k l m n o p q r s t u v w x y z
+      A B C D E F G H I J K L M N O P Q R S T U V W X Y Z
+      0 1 2 3 4 5 6 7 8 9
+      ! @ # $ % ^ & * ( \) - + = _ [ ] { } | ; : ' " < > , . / ?
+      )a
```

Comment:
Can this field be made a separate `KeyEquivalent` type? Then characters could be passed in and checked as strings or charlists over a range of characters, and the other constants could be handled as atoms.